setup.py

```diff
@@ -481,7 +481,6 @@ def check_cudnn_version_and_warn(global_option: str, required_cudnn_version: int
     for arch in [70, 75, 80, 86, 90, 100, 120]:
         arch_flag = f"-gencode=arch=compute_{arch},code=sm_{arch}"
         arch_flags.append(arch_flag)
-    arch_flag = f"-gencode=arch=compute_120,code=compute_120"
```

Comment:
This flag generates `code=compute_120`, which can be used on higher SM versions, rather than `code=sm_120`, so it's not a duplicate flag